### PR TITLE
Create Engaging with neume.md

### PR DESCRIPTION
**Pull request by djfnd**
Proposed blog

### CODE REVIEW

File: src/posts/Engaging with neume.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Discussion
+
+Currently we have three preferred communication channels:
+- On-going **text chat** within the neume channels in the [HIFI Labs discord](https://discord.gg/P5rrpZN4ds)
+    - Any and all questions around the project can be asked, across #neume-dev (technical) and #neume-chat (general)
+    - The neume core team often provide ad-hoc updates and requests to the community for discussion
+- **"neume open office hours"** audio sessions once every two weeks, held within the neume-office channel in the [HIFI Labs discord](https://discord.gg/P5rrpZN4ds)
+    - An opportunity for the community to ask specific questions and spark discussion around ideas in a voice chat format
+    - Previous open office hours are recording and saved in the #neume-achives channel on discord
```
Comment (TimDaub) [inline src/posts/Engaging with neume.md]:
```suggestion
    - Previous open office hours are recorded and saved in the #neume-achives channel on discord
```

File: src/posts/Engaging with neume.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- This **blog** where the neume core team will give regular updates on progress and direction
+
+The neume core team are also relatively active on twitter if that is your preferred format ([Tim](https://twitter.com/timdaub), [Dan](https://twitter.com/dan_djfnd_)), and we plan to ramp up activity through the [neume](https://twitter.com/neumenetwork) twitter in good time.
```
Comment (TimDaub) [inline src/posts/Engaging with neume.md]:
```suggestion
The neume core team are also relatively active on Twitter if that is your preferred format ([Tim](https://twitter.com/timdaub), [Dan](https://twitter.com/dan_djfnd_)), and we plan to ramp up activity through the [neume](https://twitter.com/neumenetwork) Twitter in good time.
```

Comment (TimDaub) [reply src/posts/Engaging with neume.md]: @il3ven wanna have a shoutout here too?

File: src/posts/Engaging with neume.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# Engaging with neume
+
+neume, as an open-source project, is dependent on the community surrounding it. As the project develops it’s important that we grow and refine the basis on which the community is encouraged to engage.
+
+## Discussion
+
+Currently we have three preferred communication channels:
+- On-going **text chat** within the neume channels in the [HIFI Labs discord](https://discord.gg/P5rrpZN4ds)
+    - Any and all questions around the project can be asked, across #neume-dev (technical) and #neume-chat (general)
```
Comment (TimDaub) [inline src/posts/Engaging with neume.md]: for some reason these indented bullet points are displayed as such on the site :(

**Review by TimDaub — APPROVED** [review]
this post is helpful to anyone new and a good exercise we should continually do to keep everyone on the same page about our organzation etc. thanks for taking care @djfnd!

File: src/posts/Engaging with neume.md (hunk 3 above)
Comment (djfnd) [reply src/posts/Engaging with neume.md]: Hmm good catch. I will play around to see if I can get indented bullets and if not then combine into a single bullet

Comment (djfnd) [reply src/posts/Engaging with neume.md]: OK applied a fix using numbers to separate. Interesting nested bullets don't seem to be supported by the theme ... https://eleventyduo.netlify.app/

**Comment by djfnd** [comment]
OK I'm happy with the formatting now. Ready to merge @TimDaub 

Some learnings:
- We can't have sub-bullets with this blog framework
- If bullet points go over 2 lines then they break
- We can't bold text int he framework

It's all good, simple is best, just something to bear in mind when writing blog.

**Pull request merged**